src/Swashbuckle.AspNetCore.SwaggerGen/Annotations/DataAnnotationAttributesOperationFilter.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Reflection;
+
+using Swashbuckle.AspNetCore.Swagger;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen
+{
+    public class DataAnnotationAttributesOperationFilter : IOperationFilter
+    {
+        public void Apply(Operation operation, OperationFilterContext context)
+        {
+            if (operation.Parameters == null)
+            {
+                return;
+            }
+
+            for (var index = 0; index < operation.Parameters.Count; index++)
+            {
+                var parameter = operation.Parameters[index];
+                if (parameter is NonBodyParameter nonBodyParam)
+                {
+                    var propertyParam = context.ApiDescription.ParameterDescriptions
+                       .Where(p => p.ModelMetadata?.ContainerType != null && p.ModelMetadata?.PropertyName != null)
+                        .FirstOrDefault(p => parameter.Name.Equals(p.Name, StringComparison.OrdinalIgnoreCase));
```

Comment:
@Erikvl87 well this is the way it is done in XmlCommentsOperationFilter already. See https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/1292e2b9f410bb6ef29c457d685241d2730330c1/src/Swashbuckle.AspNetCore.SwaggerGen/Annotations/XmlCommentsOperationFilter.cs